bears/js/ESLintBear.py

```diff
@@ -58,15 +58,17 @@ def generate_config(filename, file):
         return '{"extends": "eslint:recommended"}'
 
     def process_output(self, output, filename, file):
-        if output[1]:
-            self.warn('While running {0}, some issues were found:'
-                      .format(self.__class__.__name__))
-            self.warn(output[1])
-
         if not file or not output[0]:
             return
 
-        output = json.loads(output[0])
+        # ESLint throws error inside stdout
+        try:
+            output = json.loads(output[0])
+        except ValueError:
+            self.warn('While running {0}, some issues were found:'
+                      .format(self.__class__.__name__))
+            self.warn(output[0])
```

Comment:
btw, if you don't need `output[1]` any more, you can omit `use_stderr` and `use_stdout` in `@linter` altogether, and just use `output` on its own.

Reply:
Okay, thanks :)